system/Language/Language.php

```diff
@@ -152,14 +152,14 @@ protected function parseLine(string $line): array
 		$file = substr($line, 0, strpos($line, '.'));
 		$line = substr($line, strlen($file) + 1);
 
-		if (! array_key_exists($line, $this->language))
+		if (! isset( $this->language[$this->locale][$file] ) )
```

Comment:
This needs to stay `array_key_exists` to allow for the possibility of an empty string being the value, since that is technically valid.

Reply:
oh - yes didn't think of this possibility